util/middleware.js

```diff
@@ -10,37 +10,58 @@ export function getUserUrl (type) {
 
 /**
  * This convenience method wraps a fetch call to the specified URL
```

Comment:
This is no longer really a convenience method, but more of a private method that contains all of the auth internals. Bit of a nit, but could you change "convenience" to "internal" and change the name secureFetchCore to `secureFetchInternal` or maybe we should follow the convention of using the leading single underscore `_secureFetch`: https://stackoverflow.com/a/42350877/915811

Reply:
I'll use 'Internal' instead of underscore, as that function is not exported and is not a class member.

Reply:
Updated in https://github.com/ibi-group/otp-admin-ui/pull/33/commits/95f1c6ef3b278593aedee0c061ec187660cb617a.